### PR TITLE
Sd security 12 05

### PR DESCRIPTION
**Pull request by homelessbirds**
## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nrsg2202-01.mdx [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+
+## Timeline details [#timeline]
+
+This issue was introduced when New Relic added support for the Log Forwarding feature with Microsoft Extensions Logging (MEL) framework in .NET Core applications in .NET agent v 9.7.0 (April 4, 2022) and in .NET Framework applications in v 10.0.0 (July 19, 2022).  In version 9.7.0, the log forwarding feature was disabled by default, so customers using version 9.7.0.0 may only be affected by this issue if they have manually configured log forwarding.  
+
+In version 9.8.0 (May 5th, 2022), the Log Forwarding feature was enabled by default.  
+
+The issue was fixed with the release of .NET agent version 10.1.0, released on September 12, 2022. 
+
+## Frequently asked questions [#faq]
```
Comment (austin-schaefer) [inline src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nrsg2202-01.mdx:172]: You may need a line break here between the header and the first list element for this to render properly.

File: src/content/docs/security/security-privacy/information-security/security-bulletins.mdx [hunk 2]
```diff
@@ -43,6 +43,20 @@ Security bulletins for our [APM](/docs/apm/new-relic-apm/getting-started/introdu
   </thead>
 
   <tbody>
+    <tr>
+       . <td>
+        [NRSG22-01](/docs/security/new-relic-security/security-bulletins/security-bulletin-nrsg2202-01)
+       </td>
+
+       <td>
+        New Relic recommends that customers who deploy the .NET Agent in a configuration using Microsoft Extensions Logging (MEL) should update to version 10.1.0 or later.
```
Comment (austin-schaefer) [inline src/content/docs/security/security-privacy/information-security/security-bulletins.mdx:52]: `Agent` s/be lowercase (not a release blocker)

File: src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nrsg2202-01.mdx [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+* **Technical Links**: [Updating the .NET agent](https://docs.newrelic.com/docs/apm/agents/net-agent/installation/update-net-agent/)
+* **Workarounds**: Affected customers who cannot update their .NET agents to 10.1.0 or later at this time can [disable log forwarding](https://docs.newrelic.com/docs/logs/logs-context/disable-automatic-logging/). 
+
+## Technical details: [#technical-details]
+
+Version 10.1.0 remediates an error in the timing of the instrumentation point for Microsoft Extensions Logging to correctly send MEL data after the built-in log level filtering occurs. 
+
+## Timeline details [#timeline]
+
+This issue was introduced when New Relic added support for the Log Forwarding feature with Microsoft Extensions Logging (MEL) framework in .NET Core applications in .NET agent v 9.7.0 (April 4, 2022) and in .NET Framework applications in v 10.0.0 (July 19, 2022).  In version 9.7.0, the log forwarding feature was disabled by default, so customers using version 9.7.0.0 may only be affected by this issue if they have manually configured log forwarding.
```
Comment (austin-schaefer) [inline src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nrsg2202-01.mdx:166]: `Log Forwarding` should probably be lowercase here and elsewhere (not a release blocker)

File: src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nrsg2202-01.mdx [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+
+In version 9.8.0 (May 5th, 2022), the Log Forwarding feature was enabled by default.  
+
+The issue was fixed with the release of .NET agent version 10.1.0, released on September 12, 2022. 
+
+## Frequently asked questions [#faq]
+1. What is a Security Guidance document?
+New Relic has issued this Security Guidance document to notify customers of the need to update their software to address a software bug that, while it cannot be exploited by a third party to gain access to customer data, still has actionable security or privacy recommendations for customers.
+2. Is it possible for a third-party to exploit this issue to access  log data that is forwarded to New Relic?
+No, the issue does not allow for exposure of data to a third party. We use a comprehensive set of technical controls to support security for data we receive. For more information, see our documentation about [data security](https://docs.newrelic.com/docs/security/security-privacy/data-privacy/security-controls-privacy/) and [data encryption](https://docs.newrelic.com/docs/security/security-privacy/compliance/data-encryption/).
```
Comment (austin-schaefer) [inline src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nrsg2202-01.mdx:176]: Here and elsewhere these absolute links should be relative links (not a release blocker but should be fixed in subsequent deploy)

File: src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nrsg2202-01.mdx [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+
+This issue was introduced when New Relic added support for the Log Forwarding feature with Microsoft Extensions Logging (MEL) framework in .NET Core applications in .NET agent v 9.7.0 (April 4, 2022) and in .NET Framework applications in v 10.0.0 (July 19, 2022).  In version 9.7.0, the log forwarding feature was disabled by default, so customers using version 9.7.0.0 may only be affected by this issue if they have manually configured log forwarding.  
+
+In version 9.8.0 (May 5th, 2022), the Log Forwarding feature was enabled by default.  
+
+The issue was fixed with the release of .NET agent version 10.1.0, released on September 12, 2022. 
+
+## Frequently asked questions [#faq]
+1. What is a Security Guidance document?
+New Relic has issued this Security Guidance document to notify customers of the need to update their software to address a software bug that, while it cannot be exploited by a third party to gain access to customer data, still has actionable security or privacy recommendations for customers.
```
Comment (austin-schaefer) [inline src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nrsg2202-01.mdx:174]: I'm not sure, do these paragraphs under the list elements render correctly?

**Review by austin-schaefer — APPROVED** [review]
All comments are non-blockers for release, **except** I don't think `yarn.lock` should have been edited. You should probably revert/discard that change before merging.

File: src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nrsg2202-01.mdx [hunk 6]
```diff
@@ -0,0 +1,182 @@
+---
+title: "New Relic advises updating .NET Agent for customers employing Microsoft Extensions Logging with log forwarding enabled"
```
Comment (austin-schaefer) [inline src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nrsg2202-01.mdx:2]: `Agent` s/be lowercase (not a release blocker)

Comment (austin-schaefer) [reply src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nrsg2202-01.mdx:2]: Is this the doc title? it's very long.

File: src/content/docs/security/security-privacy/information-security/security-bulletins.mdx (hunk 2 above)
Comment (homelessbirds) [reply src/content/docs/security/security-privacy/information-security/security-bulletins.mdx:52]: Interesting, this is showing as `agent` in VS code.

File: src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nrsg2202-01.mdx (hunk 4 above)
Comment (homelessbirds) [reply src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nrsg2202-01.mdx:176]: Done

File: src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nrsg2202-01.mdx (hunk 6 above)
Comment (homelessbirds) [reply src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nrsg2202-01.mdx:2]: Yes it is, but it's the title they requested.

File: src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nrsg2202-01.mdx [hunk 7]
```diff
@@ -0,0 +1,182 @@
...
+tags:
+  - Security
+  - Security and Privacy
+  - Security bulletin
+metaDescription: "New Relic advises updating .NET Agent for customers employing Microsoft Extensions Logging with log forwarding enabled"
```
[the remark this answers is not in the document]
Comment (homelessbirds) [reply src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nrsg2202-01.mdx:7]: done

File: src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nrsg2202-01.mdx (hunk 3 above)
Comment (homelessbirds) [reply src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nrsg2202-01.mdx:166]: done

**Pull request merged**